server/repository/src/db_diesel/requisition/requisition_row.rs

```diff
@@ -117,9 +126,11 @@ impl Default for RequisitionRow {
             max_months_of_stock: Default::default(),
             min_months_of_stock: Default::default(),
             approval_status: Default::default(),
-            program_id: Default::default(),
             linked_requisition_id: Default::default(),
             is_sync_update: Default::default(),
```

Comment:
Just fyi, the reason Default::default() is there for others, is because rust analyzer code completion (automatic implementation of default trait)